src/osd/ECBackend.cc

```diff
@@ -1469,8 +1601,73 @@ void ECBackend::start_read_op(
   dout(10) << __func__ << ": started " << op << dendl;
 }
 
+void ECBackend::start_remaining_read_op(
```

Comment:
nit: can we have a comment for future developers, mentioning that this function resembles  `start_read_op()`. and can be refactored in future?